core/src/epicli/.devcontainer/Dockerfile

```diff
@@ -35,6 +36,11 @@ RUN chmod +x /config-pre.sh \
     && chmod +x ./kubectl \
     && mv ./kubectl /usr/local/bin/kubectl \
     && kubectl version --client \
+    && echo "Installing istioctl binary ..." \
```

Comment:
Also add this to the main [deployment container](https://github.com/epiphany-platform/epiphany/blob/develop/Dockerfile) in the root of the repo